### PR TITLE
Stop setting user ID for players registered by TO

### PR DESCRIPTION
**Pull request by haradan**
Also stop showing self-registration when self-registration is disabled for the tournament.

### CODE REVIEW

File: app/policies/player_policy.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class PlayerPolicy < ApplicationPolicy
+
```
Comment (plural) [inline app/policies/player_policy.rb:2]: mega nit: drop blank line.

**Review by plural — APPROVED** [review]
I was able to patch this in nicely!  That makes me super happy! Thanks for doing the fork.

Just 2 things.

File: app/policies/player_policy.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class PlayerPolicy < ApplicationPolicy
```
[the remark this answers is not in the document]
Comment (haradan) [reply app/policies/player_policy.rb:1]: It works the same way as the tournament policy. It uses Pundit: https://github.com/varvet/pundit

Do you mean add a comment to the class? What would you put in it?

File: app/policies/player_policy.rb (hunk 1 above)
Comment (haradan) [reply app/policies/player_policy.rb:2]: Done

**Pull request merged**